doc/developers/api.rst

```diff
@@ -4,7 +4,7 @@ Developer API
 A CCF application is composed of the following:
 
 - The :ref:`Application Entry Point <developers/api:Application Entry Point>` which registers the application in CCF.
-- A collection of :cpp:class:`ccf::EndpointRegistry::Endpoint`s to user HTTP requests and grouped in a single :cpp:class:`ccf::EndpointRegistry`. A :cpp:class:`ccf::EndpointRegistry::Endpoint` reads and writes to the key-value store via the :ref:`Key-Value Store API <developers/kv/api:Key-Value Store API>`.
+- A collection of :cpp:class:`ccf::EndpointRegistry::Endpoint` handling HTTP requests and grouped in a single :cpp:class:`ccf::EndpointRegistry`. A :cpp:class:`ccf::EndpointRegistry::Endpoint` reads and writes to the key-value store via the :ref:`Key-Value Store API <developers/kv/api:Key-Value Store API>`.
```

Comment:
```suggestion
- A collection of :cpp:class:`ccf::EndpointRegistry::Endpoint`s handling HTTP requests and grouped in a single :cpp:class:`ccf::EndpointRegistry`. A :cpp:class:`ccf::EndpointRegistry::Endpoint` reads and writes to the key-value store via the :ref:`Key-Value Store API <developers/kv/api:Key-Value Store API>`.
```

Reply:
Sure - had to reword slightly to avoid sphinx errors.